packages/visualizations/src/components/Table/Table.svelte

```diff
@@ -43,4 +44,10 @@
         white-space: nowrap;
         width: inherit;
     }
+
+    .vscroll-wrapper {
+        display: inline-block;
+        height: var(--vertical-scroll-height);
```

Comment:
I'd rather have that as 100% and have the container in the app fix the height. Case is, in the Studio we often define height by ratio.

This implies adding an `height: 100%; flex-wrap: no wrap; ` on the Card.

Reply:
done